homeassistant/components/advantage_air/entity.py

```diff
@@ -24,9 +31,11 @@ def __init__(self, instance, ac_key, zone_key=None):
         )
 
     @property
-    def _ac(self):
-        return self.coordinator.data["aircons"][self.ac_key]["info"]
+    def _ac(self) -> dict[str, str | int | bool]:
+        return cast(dict, self.coordinator.data["aircons"][self.ac_key]["info"])
```

Comment:
we can make this more specific than casting it, e.g., using a typeddict

Reply:
@Bre77 Can this be done at the package level?